resilience4j-ratelimiter/src/main/java/io/github/resilience4j/ratelimiter/RateLimiter.java

```diff
@@ -49,7 +50,7 @@
  * necessary until a permit is available, and then takes it. Once acquired, permits need not be
  * released.
  */
-public interface RateLimiter {
+public interface RateLimiter extends Closeable {
```

Comment:
Since not all types of rate limiters will require closing, I'd prefer that, for now, only `SemaphoreBasedRateLimiter` would expose the `close` method, and it doesn't need to be a part of the Closable interface.